bindings/python/src/PyXRootDFile.cc

```diff
@@ -580,7 +580,7 @@ namespace PyXRootD
     if ( py_timeout && PyObjToUshrt( py_timeout, &tmp_timeout, "timeout" ) )
```

Comment:
Need to also convert `PyObjToUshrt` to more appropriate function (reminder to self).

Reply:
Done.